src/components/common/YourCompany/YourCompany.vue

```diff
@@ -197,7 +197,7 @@
 
     <!-- Name Translation(s) -->
     <div
-      v-if="!isChangeFiling && !isConversionFiling"
+      v-if="isAlterationFiling || isCorrectionFiling"
```

Comment:
This is just a simplification and also more explicit: we render this component for the specified filing types, not "all types that aren't Change or Conversion" (which is risky if we add other filing types).